Apps/W1/HybridGP/app/src/Migration/Items/GPItemMigrator.codeunit.al

```diff
@@ -346,15 +346,14 @@ codeunit 4019 "GP Item Migrator"
         TempTrackingSpecification: Record "Tracking Specification" temporary;
         DataMigrationErrorLogging: Codeunit "Data Migration Error Logging";
         CreateReserveEntry: Codeunit "Create Reserv. Entry";
-        ItemJnlLineReserve: Codeunit "Item Jnl. Line-Reserve";
         ExpirationDate: Date;
     begin
         if GPItem.ItemTrackingCode = '' then
             exit;
 
         DataMigrationErrorLogging.SetLastRecordUnderProcessing(Format(GPItemTransactions.RecordId));
 
-        ItemJnlLineReserve.InitFromItemJnlLine(TempTrackingSpecification, ItemJnlLine);
+        TempTrackingSpecification.InitFromItemJnlLine(ItemJnlLine);
```

Comment:
This change was in my last PR, but for some reason wasn't merged. I had to make this change again to align with changes in BC.

Reply:
That's because the function on Tracking Specification has been moved. From version 28 you will have to call the function on the Item Jnl. Line-Reverve anyway. But I can write some CLEAN code to keep the Tracking Specification code until then.